app/(candidate)/dashboard/website/util/websiteFetch.util.js

```diff
@@ -2,6 +2,11 @@ import { apiRoutes } from 'gpApi/routes'
 import { clientFetch } from 'gpApi/clientFetch'
```

Comment:
Should we just name this file `website.util.js` now that it's got > 1 method in it now? 🤔